ui/admin/app/components/form/host-set/azure/index.hbs

```diff
@@ -127,9 +127,9 @@
     {{on 'input' (set-from-event @model 'sync_interval_seconds')}}
     as |F|
   >
-    <F.Label>{{t 'form.name.label'}}</F.Label>
+    <F.Label>{{t 'form.sync-interval.label'}}</F.Label>
     <F.HelperText>
-      {{t 'form.name.help'}}
+      {{t 'form.sync-interval.help'}}
```

Comment:
small suggestion: im wondering if instead of having "Optional" in the description, we use `@isOptional={{true}}`

Reply:
Yes totally good call, I'll update that